src/controller/logout.controller.ts

```diff
@@ -0,0 +1,12 @@
+import { Request, Response, NextFunction} from 'express'
+
+export async function logout(req: Request, res: Response, next: NextFunction) {
+    try {
+        res.clearCookie('refreshToken')
```

Comment:
the string `'refreshToken'` is now being used in 3 places, so why don't we make that string a constant in `cookie.utils.ts` then import it in the places that string is being used. They need to all match and this will prevent us from being inconsistent later on.